pom.xml

```diff
@@ -1931,18 +1928,6 @@
     </build>
 
     <profiles>
-        <!-- mockito 5.x dropped support for Java 8, but is necessary to test against Java >17 -->
-        <profile>
-          <id>pre-java-11</id>
-          <activation>
-            <jdk>(,11)</jdk>
-          </activation>
-          <properties>
-            <!-- mockito-inline was removed in mockito 5.3, but is necessary when running against mockito 4.x for Java 8 -->
-            <mockito.version>4.11.0</mockito.version>
-            <mockito.inline.artifact>inline</mockito.inline.artifact>
```

Comment:
if this is not needed anymore then those extra deps should also be removed

you could follow `mockito.inline.artifact` to remove all of them